test/e2e/angular-app/src/app/app.component.html

```diff
@@ -0,0 +1,17 @@
+<!-- src/app/app.component.html -->
+
+<link
+href="https://ok1static.oktacdn.com/assets/js/sdk/okta-signin-widget/2.5.0/css/okta-sign-in.min.css"
```

Comment:
Not needed for this PR, but we should figure out a good way to pull the version from `package.json` instead of manually updating it every time.

Reply:
This can be pulled in from node modules using the `.angular-cli.json` file, see example here: https://github.com/okta/samples-js-angular/blob/first-pass/custom-login/.angular-cli.json#L24

Reply:
I agree